cypress/support/elements/RightNav.js

```diff
@@ -57,6 +57,10 @@ class RightNav{
         this.getRightNavExpandedSpace().should('not.be.visible');
     }
 
+    openClassWorkSections(){
+        cy.get('[data-test=class-work-section]').click({force:true, multiple:true});
+    }
+
```

Comment:
General function to open all sections so related Cypress tests can access content.